src/assets/Doneth.sol.txt

```diff
@@ -49,6 +45,7 @@ contract Doneth {
         uint256 shares;
         uint256 withdrawn;
         string memberName;
+        mapping(address => uint256) tokenWithdrawn;
```

Comment:
tokenWithdrawn => tokensWithdrawn

Also, we should have a comment on what this mapping does. My assumption is that it is a mapping of an address representing an ERC20 token contract to a uint256 that represents the amount of that token withdrawn by the member?

Reply:
good call, you're right tho